karaf/init/src/main/java/org/apache/brooklyn/launcher/osgi/OsgiLauncher.java

```diff
@@ -202,8 +199,7 @@ public void setLocalBrooklynProperties(String localBrooklynProperties) {
         this.localBrooklynProperties = localBrooklynProperties;
     }
 
-    public void setDeserializingClassRenamesProvider(DeserializingClassRenamesProvider deserializingClassRenamesProvider) {
-        this.deserializingClassRenamesProvider = deserializingClassRenamesProvider;
+    public void setDeserializingClassRenamesProvider(DeserializingClassRenamesProvider.OsgiConfigLoader val) {
```

Comment:
Shouldn't it be only `DeserializingClassRenamesProvider` here? That's what the bean configuration specifies in the `blueprint.xml`